src/Marten/Linq/WhereFragments/FullTextWhereFragment.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Linq;
+using Marten.Schema;
+
+namespace Marten.Linq.WhereFragments
+{
+    internal class FullTextWhereFragment : WhereFragment
+    {
+        public FullTextWhereFragment(DocumentMapping mapping, string searchFunction, string searchTerm, string regConfig = FullTextIndex.DefaultRegConfig)
+            : base(GetFilter(mapping, searchFunction, searchTerm, regConfig))
+        {
+        }
+
+        private static string GetFilter(DocumentMapping mapping, string searchFunction, string searchTerm, string regConfig)
+        {
+            var dataConfig = GetDataConfig(mapping, regConfig);
+
+            return $"to_tsvector('{regConfig}', {dataConfig}) @@ {searchFunction}('{regConfig}', '{searchTerm}')";

```

Comment:
Sql injection possible?

Reply:
Good catch (and shame on me for being too hasty)!. It is. `x.Search(@"somefilter ') ; drop table mt_doc_blogpost ; select ('"))`.

Reply:
Yeah, I forgot about that. I was supposed to fix that issue https://github.com/JasperFx/marten/issues/1170. I'll work on that

Reply:
@mdissel @jokokko  @RobSchoenaker Fixed in https://github.com/JasperFx/marten/pull/1167/commits/b31120cbd2b91844f3905bed69be998a6bd5908a